README.md

```diff
@@ -1,45 +1,117 @@
 
+---
+**Project Task Requirements:**
+
+<ins>Minimal Requirements:</ins>
+* Front end page to view list of courses -   :heavy_check_mark:
+* Back end that takes user input and stores data -   :heavy_check_mark:
+* Database to store course reviews added by users -   :heavy_check_mark:
+
+<ins>Standard Requirements:</ins>
+* Front end page for each individual course -   :heavy_check_mark:
+* Add ratings for each course -   :heavy_check_mark:
+* Search for courses by course code -   :heavy_check_mark:
+* Filter for courses based on year level and  -   :heavy_check_mark:
+* Contact form/Button to report issues -   :heavy_check_mark:
+
+<ins>Stretch Requirements:</ins>
+* Add historical data for each course -   :x:
+* User Academic Profile (i.e. Computer Science 2nd Year) -   :x:
+* Display user's comment history -   :x:
+
+---
+
+**Required Tech Stack:**
+* **HTML, CSS, JS**
+	We used in various parts of our project to create divs and sometimes paragraph tags for formatting. CSS is used for all of our components for styling, formatting, and controlling our website appearance. Javascript is used across all or most of our project components to add functionality to the front end such as mapping and filtering.
+
+* **React & Redux**
+	We used React for our front end. Every piece of our website is made using React components, with Material UI for styling. Redux is used in our project to manage the states of our application through actions and reducers.
+
+* **MongoDB**
+	We used MongoDB Atlas for our database to store our data on courses, reviews, and users. Whenever our web app needs data, it will get the relevant data from our collection in MongoDB to display.
+
+* **NodeJS & Express**
+	We used NodeJS for our server functionalities and Express framework to for our REST API endpoints. We implemented multiple routes to perform CRUD operations to the back end.
+
+* **Heroku**
+We are using Heroku to host our web app and our CD pipeline. Every time a pull request is made, it will be automatically deployed to Heroku.
+
+---
+**Additional Features:**
+* **Pages**
+Because the list of courses that we have on the homepage may get too long for a user-friendly experience, we decided to implement pages of 10 courses per page. The courses on each page is fetched from the back end to reduce the time spent grabbing the entire list of courses from the database.
+
+* **Role-Based Routing**
```

Comment:
Minor nit: Role-based-access-control would be better suited here IMO - since it also conditionally renders front end components in addition to the back-end routing.

Reply:
OK lemme change that